storage/localstore/localstore.go

```diff
@@ -80,6 +80,9 @@ type DB struct {
 	// garbage collection index
 	gcIndex shed.Index
 
+	// pin files Index
+	pinIndex shed.Index // Index which stores address of all the pinned chunks and their pin counter
```

Comment:
remove the inlined comment (this is clear from reading 343-344)

Reply:
Done